majasdarbs-05-2022/src/io/codelex/arithmetic/practice/numbersix.java

```diff
@@ -0,0 +1,43 @@
+package io.codelex.arithmetic.practice;
+
+import java.util.stream.IntStream;
+
+public class numbersix {
+
+    public static void main(String[] args) {
+        CozaLozaWoza();
+    }
+
+    public static void CozaLozaWoza() {
```

Comment:
Šis fails arī jānoformatē, kā arī neatstāj tik daudz tukšas rindas pa vidu. Turpmāk par formatting nekomentēšu - vienkārši pārej pāri failiem un noformatē.